src/firebase/utils/batchUpdateData.js

```diff
@@ -0,0 +1,68 @@
+import db from '../firebase';
+import { awardConsiderationEnum } from '../../constants';
+
+const getIndex = (action, performanceDocDataEnum) => {
+  const { DECREMENT } = awardConsiderationEnum;
```

Comment:
I'm curious if we could also get some documentation on this. I'm confused how this IDX logic below works in calculating an index.

Reply:
@YangStephen  can do this in a separate PR this weekend